### ISSUE TITLE
Add alt text

### ISSUE THREAD

**Issue by atsansone** [dart-lang/site-www#5300]
Add `alt` attribute to all images on the site. This violates WCAG [H37](https://www.w3.org/TR/WCAG20-TECHS/H37).

Refer to [the W3C’s alt Decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/) and [Axess Lab’s Ultimate Guide to Alt Texts](https://axesslab.com/alt-texts/) for guidance.

See #1104

**Comment by atsansone** [comment]
Ran [this tool](https://www.lookkle.com/web-analytics/website-broken-link-scan.php?website=https%3A%2F%2Fdart.dev) and this regex (`(<img(?!.*?alt=(['"]).*?\2)[^>]*)(>)` to find all `<img>` tags without the `alt` attribute.

Both returned no files. Closing.
